### DISCUSSION TITLE
metad: Can VolumeClaim be used for both log & data

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4297] [Q&A]
Version: 3.1.0
Nebula Helm chats

- Currently, separate LOG & DATA volume claims is MUST for metad
- We use one Volume claim for both?

**Comment by wey-gu** [comment]
Is it feasible for now to use one volume for both data and log? @MegaByte875 if I recall it correctly, it was possible for an older version of the operator :)
